src/test/xml_coverage.go

```diff
@@ -51,13 +52,13 @@ func coverageResultToXML(sources []core.BuildLabel, coverage core.TestCoverage)
 	// get the string representative of sources
 	var sourcesAsStr []string
 	for _, source := range sources {
-		sourcesAsStr = append(sourcesAsStr, source.String())
+		sourcesAsStr = append(sourcesAsStr, path.Join(core.RepoRoot, "/", source.PackageName))
```

Comment:
you don't need the "/" if you're using JoinPath - let it add it for you if necessary.